assets/js/components/settings/SettingsAdmin.js

```diff
@@ -24,24 +24,50 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import { MODULES_SEARCH_CONSOLE } from '../../modules/search-console/datastore/constants';
+import { MODULES_ANALYTICS_4 } from '../../modules/analytics-4/datastore/constants';
 import Layout from '../layout/Layout';
 import { Grid, Cell, Row } from '../../material-components';
 import OptIn from '../OptIn';
 import ResetButton from '../ResetButton';
 import { useFeature } from '../../hooks/useFeature';
 import SettingsCardKeyMetrics from './SettingsCardKeyMetrics';
 import SettingsPlugin from './SettingsPlugin';
+const { useSelect } = Data;
 
 export default function SettingsAdmin() {
 	const userInputEnabled = useFeature( 'userInput' );
 
+	const searchConsoleModuleConnected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'search-console' )
+	);
```

Comment:
This shouldn't be necessary since the module cannot be deactivated and is required for SK to be considered set up which is a requirement to be able to reach the settings.

The module will actually always be considered connected since it does not implement its own logic for `is_connected` – the "connected state" is essentially enforced through the setup-completion of SK itself without which, most of our capabilities are not granted.

Reply:
Thanks for the reminder here. A check for the Search Console connected state had made its way into the `KeyMetricsSetupCTAWidget` component which I was referring to and the fact SC can't be disconnected had slipped my mind.

I'll remove this check, and the one in `KeyMetricsSetupCTAWidget` for good measure.